frontend/src/scenes/me/Settings/UserDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react'
+import { Input, Form } from 'antd'
+import { useActions, useValues } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+
+export function UserDetails(): JSX.Element {
+    const { user, userLoading } = useValues(userLogic)
+    const { updateUser } = useActions(userLogic)
+    const [form] = Form.useForm()
+
+    useEffect(() => {
+        form.setFieldsValue({
+            first_name: user?.first_name,
+        })
+    }, [user?.first_name])
+
+    return (
+        <Form
+            onFinish={(values) => updateUser(values)}
+            labelAlign="left"
+            layout="vertical"
+            requiredMark={false}
+            form={form}
+            style={{
+                maxWidth: 400,
+            }}
+        >
+            <Form.Item
```

Comment:
The "[how to build a form](https://storybook.posthog.net/?path=/docs/how-to-build-a-form--page)" page in storybook lists an alternative to this antd form approach, which we're trying to stop using.